### PR TITLE
Debugging - MRT overwriting/removing previous include\* when published MRM.h?

### PR DESCRIPTION
**Pull request by DrusTheAxe**

### CODE REVIEW

**Comment by DrusTheAxe** [comment]
Resulting transport package (FullNuget.zip) now includes the previously missing files :-)

File: build/CopyFilesToStagingDir.ps1 [hunk 1]
```diff
@@ -82,27 +82,45 @@ PublishFile $FullBuildOutput\ProjectReunion_DLL\Microsoft.ProjectReunion.pdb $sy
...
+# MSIX Framework package - DLLs
+PublishFile $FullBuildOutput\ProjectReunion_DLL\Microsoft.ProjectReunion.dll $NugetDir\runtimes\win10-$Platform\native
+PublishFile $FullBuildOutput\ProjectReunion_DLL\Microsoft.ProjectReunion.pdb $NugetDir\runtimes\win10-$Platform\native
+#
+# MSIX Main package
 PublishFile $FullBuildOutput\DynamicDependency.DataStore\DynamicDependency.DataStore.exe $NugetDir\runtimes\win10-$Platform\native
-PublishFile $FullBuildOutput\DynamicDependency.DataStore\DynamicDependency.DataStore.pdb $NugetDir\runtimes\win10-$Platform\native\
+PublishFile $FullBuildOutput\DynamicDependency.DataStore\DynamicDependency.DataStore.pdb $NugetDir\runtimes\win10-$Platform\native
 PublishFile $FullBuildOutput\DynamicDependency.DataStore.ProxyStub\DynamicDependency.DataStore.ProxyStub.dll $NugetDir\runtimes\win10-$Platform\native
 PublishFile $FullBuildOutput\DynamicDependency.DataStore.ProxyStub\DynamicDependency.DataStore.ProxyStub.pdb $NugetDir\runtimes\win10-$Platform\native\
```
Comment (kythant) [inline build/CopyFilesToStagingDir.ps1:106]: Did you also want to remove the \ at the end here?

File: build/publish-mrt.yml [hunk 2]
```diff
@@ -128,6 +128,10 @@ steps:
     TargetFolder: '$(Build.ArtifactStagingDirectory)\fullnuget\build'
     flattenFolders: false
 
+- script: |
```
Comment (kythant) [inline build/publish-mrt.yml:131]: A better display name for these we if we are keeping these in?

File: build/publish-mrt.yml [hunk 3]
```diff
@@ -136,11 +140,19 @@ steps:
       mrm.h
     TargetFolder: '$(Build.ArtifactStagingDirectory)\fullnuget\include'
 
+- script: |
+    dir /s $(Build.ArtifactStagingDirectory)\fullnuget
+  displayName: HK 2
```
Comment (kythant) [inline build/publish-mrt.yml:145]: Here

File: build/publish-mrt.yml [hunk 4]
```diff
@@ -136,11 +140,19 @@ steps:
...
+
 - task: PublishBuildArtifacts@1
   inputs:
     PathtoPublish: '$(Build.ArtifactStagingDirectory)\fullnuget'
     ArtifactName: 'FullNuget'
 
+- script: |
+    dir /s $(Build.ArtifactStagingDirectory)\fullnuget
+  displayName: HK 3
```
Comment (kythant) [inline build/publish-mrt.yml:154]: Here

File: build/CopyFilesToStagingDir.ps1 (hunk 1 above)
Comment (DrusTheAxe) [reply build/CopyFilesToStagingDir.ps1:106]: Thought I'd got all those. Thanks!

File: build/publish-mrt.yml (hunk 2 above)
Comment (DrusTheAxe) [reply build/publish-mrt.yml:131]: Snicker. These should have been commented out. Thanks.

I'm planning a future PR removing all these (and other commented-out dead code) and add a separate README.md or something that includes these (and other) debugging tips.

File: build/publish-mrt.yml (hunk 3 above)
Comment (DrusTheAxe) [reply build/publish-mrt.yml:145]: Yup. See other comment. Thanks

File: build/publish-mrt.yml (hunk 4 above)
Comment (DrusTheAxe) [reply build/publish-mrt.yml:154]: Yup 3rd of the trinity. See other comment. Thanks

File: test/DynamicDependency/data/DynamicDependencyLifetimeManagerGC1010.Msix/appxmanifest.xml [hunk 5]
```diff
@@ -45,7 +45,7 @@
       </uap:VisualElements>
       <Extensions>
         <uap3:Extension Category="windows.appExtension">
-          <uap3:AppExtension Name="com.microsoft.reunion.ddlm.4.1.###architecture###"
+          <uap3:AppExtension Name="com.microsoft.reunion.ddlm-4.1-###architecture###"
```
Comment (kythant) [inline test/DynamicDependency/data/DynamicDependencyLifetimeManagerGC1010.Msix/appxmanifest.xml:48]: There is alot of tidbits like this I am not understanding in general but I trust it's there for a reason. Like why is there a hardecoded 4.1 in the name?

**Review by kythant — APPROVED** [review]
But someone else should review and approve as well hahah

File: test/DynamicDependency/data/DynamicDependencyLifetimeManagerGC1010.Msix/appxmanifest.xml (hunk 5 above)
Comment (DrusTheAxe) [reply test/DynamicDependency/data/DynamicDependencyLifetimeManagerGC1010.Msix/appxmanifest.xml:48]: These are test packages. They use constant data. Real builds use build input and even dynamically generate certain values (e.g. DDLM's CLSID) via build\override\dynamicdependencies-override.json

Any packages under test/* are by definition 'fake' as they cannot use Project Reunion published MSIXs, only local packages for F5/unit testing.

The 'real' Project Reunion MSIXs are used by Integration Tests, which by definition are built and run with artifacts of the end of the pipeline (the aggregator).

**Pull request merged**